components/listingsPage/filters.js

```diff
@@ -91,6 +91,41 @@ const Filters = ({ t }) => {
 
   const offerings = useSelector((state) => state.listings.offerings.hits);
 
+  // google tagging
+
+  if (process.browser) {
+    window.dataLayer = window.dataLayer || [];
+  }
+  const tag = (e) => {
+    let y;
+
+    let x = e.target.value;
+
+    if (e.target.name === "age" && x >= 18 && x <= 24) {
+      y = "18-24";
+    } else if (e.target.name === "age" && x >= 25 && x <= 34) {
+      y = "25-34";
+    } else if (e.target.name === "age" && x >= 35 && x <= 44) {
+      y = "35-44";
+    } else if (e.target.name === "age" && x >= 45 && x <= 54) {
+      y = "45-54";
+    } else if (e.target.name === "age" && x >= 55 && x <= 64) {
+      y = "55-64";
+    } else if (e.target.name === "age" && x >= 65 && x <= 74) {
+      y = "65-74";
+    } else if (e.target.name === "age" && x >= 74) {
+      y = "75+";
+    } else {
+      y = x;
+    }
+
+    window.dataLayer.push({
+      event: "travelDirectory_RadioButton",
+      buttonName: e.target.name,
+      buttonChoice: y,
+    });
```

Comment:
This looks perfect for the radio buttons but I think we need to record a slightly different set of data values for the age range option: the `buttonName` and `buttonChoice` values are replaced in this instance by `ageRange`.

Reply:
Done